command/status.go

```diff
@@ -0,0 +1,135 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+type StatusCommand struct {
+	Meta
+}
+
+func (c *StatusCommand) Run(args []string) int {
+	// Get the HTTP client
+	client, err := c.Meta.Client()
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error initializing client: %s", err))
+		return 1
+	}
+
+	// Parsing args is not idempotent
+	argsCopy := args
```

Comment:
Can you put this next to line 34 and I think the comment is slightly misleading. Maybe "Store the original arguments so we can pass them to the routed command".